src/components/TodoList.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable react/prop-types */
```

Comment:
Kindly avoid disabling the linters because linters make your code clean and follow standards.

Reply:
I will get the error fixed.